### PR TITLE
use class syntax in stream-view

### PR DESCRIPTION
**Pull request by batfink**
Since using util.inherits is now [discouraged](https://nodejs.org/docs/latest/api/util.html#util_util_inherits_constructor_superconstructor), I thought I’d rewrite the [stream-view example](https://github.com/koajs/examples/tree/master/stream-view) using the class syntax as part of my own learning process.

### CODE REVIEW

**Comment by batfink** [comment]
- The Travis build fails because the version of node used to run the test is outdated. 
- Snyk complains about uglify.js

Fixing these issues is not within my jurisdiction I’m afraid.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 node_js:
-- "0.12"
+- "6.3"
```
Comment (dead-horse) [inline .travis.yml]:
```
node_js:
  "4"
  "6"
```

remove iojs and add node 4

**Comment by batfink** [comment]
@dead-horse: Done. I also added 'use strict' to view.js (node 4 requirement) and removed the `--harmony` flag from the README since it’s no longer necessary. Rebased to a single commit.

**Comment by dead-horse** [comment]
👍

**Pull request merged**